### PR TITLE
add runtime status test case

### PR DESCRIPTION
**Pull request by Helen-Xie**
part of #4 

Signed-off-by: Helen-Xie <yulinxie@zju.edu.cn>

### CODE REVIEW

File: pkg/validate/status.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import (
+	"github.com/kubernetes-incubator/cri-tools/pkg/framework"
+	internalapi "k8s.io/kubernetes/pkg/kubelet/api"
+
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = framework.KubeDescribe("Version", func() {
```
Comment (feiskyer) [inline pkg/validate/status.go]: s/Version/Status

File: pkg/validate/status.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+		for _, condition := range status.Conditions {
+			if condition.Type == "RuntimeReady" && condition.Status == true {
+				count = count + 1
+			}
+			if condition.Type == "NetworkReady" && condition.Status == true {
+				count = count + 1
+			}
+		}
+		Expect(count).To(Equal(2), "should return all the required runtime cinditions")
```
Comment (feiskyer) [inline pkg/validate/status.go]: Should be `>=2`, runtime should return at least two conditions

Comment (feiskyer) [inline pkg/validate/status.go]: s/cinditions/conditions

**Comment by Helen-Xie** [comment]
Update @feiskyer

**Comment by feiskyer** [comment]
LGTM. Thanks.

**Pull request merged**